codeforboston/vaccinatema
# Update README and delete unused assets.

Now that we're finished porting the frontend to React, we can update the README and delete some unused files. (The map pins are now SVGs.)

### Test Plan:
* Ensure the map pins load as usual.
* Ensure no console errors.
